### DISCUSSION TITLE
Close issues opened before 2021?

### DISCUSSION THREAD

**Discussion by casch-at** [bastibe/org-journal#444] [Ideas]
So much has changed since.

**Comment by bastibe** [comment]
Good idea. Mark them as abandoned and outdated, and move on.

In general, I require at least two interactions of issue reporters before doing anything on my end. Just asking for something does not demonstrate a concrete need. Only if the asker exerts some effort themselves, or a second person chimes in, do they prove that it's more than a drive-by comment.

Hence the many abandoned issues.

**Comment by casch-at** [comment]
perfect :-) lets use the label "close-me" to mark the relevant issues before we (!)close them
